// Copyright 2015 gRPC authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

option objc_class_prefix = "AU";

package author;

// The greeting service definition.
service Author {
  // Sends a greeting
  rpc GetAuthor (AuthorRequest) returns (AuthorReply) {}
}

// The request message containing the user's name.
message AuthorRequest {
  int32 id = 1;
}

// The response message containing the greetings
message AuthorReply {
  int32 id = 1;
  string first_name = 2;
  string last_name  = 3;
  string email      = 4;
  string title      = 5;
}
